sdk/identity/perf-tests/identity/test/ClientSecretCredential/persistence.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+import { ClientSecretCredential } from "@azure/identity";
+
+const scope = `https://servicebus.azure.net/.default`;
+
+/**
+ * This test does silent authentication with persistence enabled.
+ */
+export class ClientSecretCredentialPersistenceTest extends PerfStressTest {
+  options = {};
+  credential: ClientSecretCredential;
+
+  constructor() {
+    super();
+    const tenantId = process.env.AZURE_TENANT_ID!;
+    const clientId = process.env.AZURE_CLIENT_ID!;
+    const clientSecret = process.env.AZURE_CLIENT_SECRET!;
```

Comment:
You can use `getEnvVar()` from the framework, it'd throw if not provided.

Reply:
cool! I'll use it

Reply:
also, using `!` would not pass linting. Please make sure `rushx lint` succeeds because it is not currently checked in CI for perf tests.

Reply:
lint passes now :)